src/DynamoCore/Library/DocumentationServices.cs

```diff
@@ -76,14 +76,33 @@ public static XmlReader GetForAssembly(string assemblyPath, IPathManager pathMan
             if (File.Exists(documentationPath))
                 return true;
 
+            //try with uppercase XML for linux
+            documentationPath = PathHelper.replaceExtension(documentationPath, ".XML");
```

Comment:
Agree with Mike, there seems to be built in priority according to the sequence of code. Does not look like that should be wrapped as well.